cache/edge_lambdas/redirects.js

```diff
@@ -162,4 +162,6 @@ module.exports = {
   '/installations/Wrynhx8AAAjk9XX-': '/exhibitions/XFximBAAAPkAioW1',
   '/articles/xksu0xiaadlrl4-h': '/articles/XKsU0xIAADlrL4-h',
   '/landing-pages/X2jSjBMAACIA8Wq_': '/collections',
+  '/pages/X5amzBIAAB0Aq6Gm': '/covid-welcome-back',
+  '/pages/X5aomxIAAB8Aq6n5': '/covid-book-your-ticket',
```

Comment:
We use english URLs in a few places, should we build something that covers all of them?

Reply:
I see we've done this for `/collections` too - but it doesn't feel massively sustainable.

Reply:
I think the only reason these have vanity urls is because they weren't able to be Prismic-ed in the first instance, not because there was demand for it. So I don't necessarily think this is something we need to worry about happening often.

Reply:
Should we not be redirecting the other way round then?